src/templates/blog.tsx

```diff
@@ -1,10 +1,15 @@
 import React, { useMemo } from 'react';
 import { injectIntl, WrappedComponentProps } from 'react-intl';
 import DefaultLayout from '../layouts/default';
-import { BlogComponents, ArticleComponents } from '../components';
+import {
+  ArticleComponents,
+  BlogComponents,
+  CommonComponents,
```

Comment:
I think Pagination should be within `BlogComponents` as it is only something for Blog pages, no?